Source/WebCore/mathml/MathMLPresentationElement.h

```diff
@@ -60,7 +60,7 @@ class MathMLPresentationElement : public MathMLElement {
     RenderPtr<RenderElement> createElementRenderer(RenderStyle&&, const RenderTreePosition&) override;
     bool isPresentationMathML() const final { return true; }
 
-    static Length parseNumberAndUnit(StringView);
+    static Length parseNumberAndUnit(StringView, bool coreMathMLEnabled);
```

Comment:
Is coreMathMLEnabled accurate? Is it not more a question of support mode for unitless/named spaces? I think the name could be better.

Reply:
Right, I guess I can rename.